rest-api-spec/src/main/resources/rest-api-spec/api/ml.start_deployment.json

```diff
@@ -0,0 +1,30 @@
+{
+  "ml.start_deployment":{
```

Comment:
Doesn't mention "trained model` in the API name, would have to be discovered via parameter names. Perhaps `ml.[start|stop]_trained_model_deployment`?

Reply:
Renamed ✔️